addons/mail/static/src/components/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -1974,6 +1981,47 @@ QUnit.test('focusing a chat window of a chat should make new message separator d
     );
 });
 
+QUnit.test('chat window: click on correspondent name in chat header, redirects to user form', async function (assert) {
+    assert.expect(4);
+
+    this.data['res.partner'].records.push({ id: 7, name: "Demo" });
+    const chat = {
+        channel_type: "chat",
+        is_minimized: true,
+        members: [this.data.currentPartnerId, 7],
+    };
+    this.data['mail.channel'].records.push(chat);
+
+    const PartnerFormPromise = testUtils.makeTestPromise();
+    const bus = new Bus();
+    bus.on('do-action', null, payload => {
+        assert.step('action_open_partner_form');
+        assert.deepEqual(payload.action, {
+            res_id: 7,
+            res_model: "res.partner",
+            type: "ir.actions.act_window",
+            views: [[false, "form"]]
+        }, 'should open partner form');
+        return PartnerFormPromise.resolve();
+    });
+
+    await this.start({env: { bus }});
```

Comment:
space inside inline object definition
```suggestion
    await this.start({ env: { bus } });
```